app/eth2wrap/version.go

```diff
@@ -81,7 +81,6 @@ func checkBeaconNodeVersionStatus(bnVersion string) (beaconNodeVersionStatus Bea
 func CheckBeaconNodeVersion(ctx context.Context, bnVersion string) {
 	status, currentVersion, minVersion := checkBeaconNodeVersionStatus(bnVersion)
 
-	//nolint:revive // enforce-switch-style: the list is exhaustive and there is no need for default
```

Comment:
I had the same edits locally... I discarded these (version.go, exit.go, test.go), probably our local version is different to the CI version.